contracts/Token.sol

```diff
@@ -11,6 +11,9 @@ import 'openzeppelin-solidity/contracts/token/ERC20/PausableToken.sol';
 contract Token is CappedToken, PausableToken {
 
   uint256 private constant TOKEN_CAP = 980 * 10**24;
+  
+  uint256 public constant PRIVATE_SALE_CAP = 140 * 10**24;
+  uint256 public constant ICO_SALE_CAP = 448 * 10**24;
```

Comment:
Suppose it would be better to save these in ico contract - since this has nothing todo with token in the long run. but this will be ok

Reply:
Moved constants to ICO contract.